site/content/installation/ngf-images/index.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "NGINX Gateway Fabric Images"
+description:
+weight: 200
+linkTitle: "Installing NGINX Gateway Fabric"
+menu:
```

Comment:
This index.md file needs to be named _index.md.

Right now the new documents aren't actually traversable. 

You can check out other existing folders to see how Hugo structures and parses folders.

Reply:
yes I was wondering about that. Thank you. Updating